portal/src/OnboardingRedirect.tsx

```diff
@@ -3,11 +3,13 @@ import { useNavigate } from "react-router-dom";
 import ShowLoading from "./ShowLoading";
 import ShowError from "./ShowError";
 import { useAppListQuery } from "./graphql/portal/query/appListQuery";
+import { useViewerQuery } from "./graphql/portal/query/viewerQuery";
 
 const OnboardingRedirect: React.VFC = function OnboardingRedirect() {
   const { loading, error, apps, refetch } = useAppListQuery();
 
   const navigate = useNavigate();
+  const { viewer } = useViewerQuery();
```

Comment:
We need to handle `loading` of viewer query. In the `useEffect`, do not do the redirect before viewer is loaded.